app/models/manageiq/providers/azure/cloud_manager/refresh_parser.rb

```diff
@@ -87,7 +94,7 @@ def get_availability_zones
       def get_stacks
         # deployments are realizations of a template in the Azure provider
         # they are parsed and converted to stacks in vmdb
-        deployments = gather_data_for_this_region(@tds)
+        deployments = gather_data_for_this_region(@tds, 'list')
```

Comment:
@bzwei But it doesn't have a location, and the logic filters the results on the ems region.